cli/flox/src/commands/list.rs

```diff
@@ -159,8 +167,14 @@ impl List {
                 broken = broken.map(|b|b.to_string()).as_deref().unwrap_or("N/A"),
             };
 
-            println!("{message}");
+            // add an empty line between packages
+            if idx < packages.len() - 1 {
+                writeln!(&mut out, "{message}")?;
+            } else {
+                write!(&mut out, "{message}")?;
+            }
```

Comment:
note: probably worth mentioning a behavior change like this in the PR description

Reply:
the behavior does not change
it just fixes that the last line does not end with 2 new lines, do you think that's note worthy?

Reply:
Yes. It makes it easier to skim the diff if I'm not wondering why something was refactored. And it's a user facing behavior change. It might even be deliberate - we do the same thing for e.g. `flox search hello`, which has the double trailing newline to make it easier to see where the command output ends